cli/cmd/plugin/ui/web/tanzu-ui/src/components/aws/credential-step/CredentialProfile.tsx

```diff
@@ -0,0 +1,113 @@
+// React imports
+import React, { ChangeEvent, useEffect, useState } from 'react';
+
+// Library imports
+import { CdsControlMessage, CdsFormGroup } from '@cds/react/forms';
+import { CdsSelect } from '@cds/react/select';
+
+// App imports
+import { AwsService } from '../../../swagger-api/services/AwsService';
+import { UseFormReturn } from 'react-hook-form';
+import { FormInputs } from './Credentials';
+import './CredentialProfile.scss';
+
+interface Props {
+    initialProfile: string;
+    initialRegion: string;
+    regions: string[];
+    handleSelectProfile: (profile: string) => void;
+    handleSelectRegion: (region: string) => void;
+    methods: UseFormReturn<FormInputs, any>;
+}
+
+function CredentialProfile(props: Props) {
+    const {
+        methods: {
+            formState: { errors },
+            register,
+            setValue,
+        },
+        regions,
+        initialProfile,
+        initialRegion,
+        handleSelectProfile,
+        handleSelectRegion,
+    } = props;
+    const [profiles, setProfiles] = useState<string[]>([]);
+    useEffect(() => {
+        // fetch profiles
+        AwsService.getAwsCredentialProfiles().then((data: string[]) =>
+            setProfiles(data)
+        );
+        setValue('REGION', initialRegion);
+    }, []);
+
+    const handleProfileChange = (event: ChangeEvent<HTMLSelectElement>) => {
+        handleSelectProfile(event.target.value);
+    };
+
+    const handleRegionChange = (event: ChangeEvent<HTMLSelectElement>) => {
+        setValue('REGION', event.target.value, { shouldValidate: true });
+        handleSelectRegion(event.target.value);
+    };
+
+    return (
+        <>
+            <p className="description">
+                Select an already existing AWS credential profile. The access
+                keys and session token information configured for your profile
+                will be temporarily passed to the installer.
+            </p>
+            <p className="description">
+                Don&apos;t have a AWS Credential profile? Credential profiles
```

Comment:
please change "Don't have a" to "Don't have an"

Reply:
fixed